packages/lambdas/src/document-downloader.ts

```diff
@@ -145,7 +156,7 @@ export const handler = Sentry.AWSLambda.wrapHandler(
             .putObject({
               Bucket: bucketName,
               Key: fileInfo.fileName,
-              Body: newXML,
+              Body: newXmlStr,
               ContentType: "application/xml",
             })
```

Comment:
I know this is not new, but I'm thinking it would be great to store the original XML w/ a custom suffix, like `_orginal`.

This would allow us to peek into the original file should anything go wrong w/ our code (not only current version, but also future changes).

Thoughts?

Reply:
So the whole point was to limit b64 being uploaded but if would like the original file i just wont override ride it and remove the nonxml